scripts/nlp_language_modeling/preprocess_data_for_megatron.py

```diff
@@ -18,9 +18,11 @@
 import gzip
 import json
 import multiprocessing
+import os
 import sys
 import time
 
+import ftfy
```

Comment:
Do we need to add this to requirements_nlp.txt?

Reply:
Done!